users/vieting/experiments/switchboard/transducer/feat/experiments.py

```diff
@@ -277,6 +283,182 @@ def run_rasr_gt_stage1():
     return nn_system, report
 
 
+def run_scf_stage1():
+    ctc_alignment = get_ctc_alignment(use_scf_aligment=True)
+
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/transducer/feat/"
+    _, dev_corpora, _ = get_switchboard_data()
+    returnn_datasets = get_returnn_datasets_transducer_viterbi(context_window={"classes": 1, "data": 121})
+    returnn_datasets_hash_break = get_returnn_datasets_transducer_viterbi(
+        context_window={"classes": 1, "data": 121},
+        keep_hashes=False,
+    )
+    returnn_datasets_align_ctc = get_returnn_datasets_transducer_viterbi(
+        alignment=ctc_alignment,
+        features="waveform_pcm",
+    )
+    returnn_args = {
+        "batch_size": 15000,
+        "datasets": returnn_datasets,
+        "extra_args": {
+            # data sequence is longer by factor 4 because of subsampling and 80 because of feature extraction vs.
+            # raw waveform
+            "chunking": ({"classes": 64, "data": 64 * 4 * 80}, {"classes": 32, "data": 32 * 4 * 80}),
+            "gradient_clip": 20.0,
+            "learning_rate_control_error_measure": "sum_dev_score",
+            "min_learning_rate": 1e-6,
+        },
+        "specaug_old": {"max_feature": 15},
+    }
+    returnn_args_ctc_align = copy.deepcopy(returnn_args)
```

Comment:
You don't have any experiments that do not use the CTC alignment from SCF, so you can directly create the suitable `returnn_args` and don't need to modify them here.